src/cmd/run.js

```diff
@@ -256,6 +273,19 @@ export default async function run(
         );
       }
 
+      readline.emitKeypressEvents(process.stdin);
+      if (process.stdin.isTTY) {
+        log.info('Press R to reload');
+        (process.stdin: any).setRawMode(true);
```

Comment:
This `(process.stdin: any).setRawMode(...)` looks like something that is hiding a flow validation error, am I right?

we should try to fix the flow validation error properly (and not by using the `any` type workaround which is too broad), e.g. it seems that the issue could be that flow is not able to know if process.stdin is going to be a tty.ReadStream (https://nodejs.org/api/tty.html#tty_class_tty_readstream), and so my guess is that something like this should make flow happy:

```
if (process.stdin instanceof tty.ReadStream) {
  process.stdin.setRawMode(true);
}
```